security-force-monitor/sfm-cms
# Small tweaks to display text

## Overview

In the course of QA for the [CRUD bulletproofing project](https://github.com/security-force-monitor/sfm-cms/projects/4), we identified a number of cases of display text that needed to be tweaked. This PR batches these tweaks into a single change for efficiency.

Connects #633, #420, #446.

### Notes

For clarity, this PR represents the following changes:

- Change the deletion warning that prompts users to delete related entities
- Update Posting and Personnel Create views to use the word "Add" in the form title
- Remove Edit icon from Person detail view 

## Testing Instructions

* Confirm new tests pass on Travis
* Navigate to http://localhost:8000/en/organization/delete/fea879cb-c680-4496-ba75-6f58fecf218d/ and confirm the deletion text has been updated
* Confirm the word "Add" is in the form title of the following Create views:
    * http://localhost:8000/en/organization/create/personnel/fea879cb-c680-4496-ba75-6f58fecf218d/
    * http://localhost:8000/en/person/create/posting/c42ab2f7-6fcd-4aa5-a106-60dcaba7b72b/
* Visit http://localhost:8000/en/person/view/c42ab2f7-6fcd-4aa5-a106-60dcaba7b72b/ and confirm that no pencil icon displays in the `Postings` table
    * Conversely, visit http://localhost:8000/en/person/create/posting/c42ab2f7-6fcd-4aa5-a106-60dcaba7b72b/ and confirm the pencil icon _does_ display
